app/errors/pending_data_migration_error.rb

```diff
@@ -0,0 +1,6 @@
+# Rails 6 has ActiveSupport::ActionableError which would allow us to tell the
```

Comment:
👍 , I think I also suggested this in one of our issues, would be great to get to Rails 6 😃